keyboards/atreus/keymaps/replicaJunction/keymap.c

```diff
@@ -1,78 +1,79 @@
-/*
- * Keyboard: Atreus
- * Keymap:   replicaJunction
- * Version:  2.1
- */
-
 #include QMK_KEYBOARD_H
```

Comment:
Could you add a GPL2+ compatible license header here? 

For instance: 
https://github.com/qmk/qmk_firmware/blob/28ff51175b6d45ce5a8dbfe7ed6e7a12df3bc8a8/quantum/template/base/keyboard.c#L1-L15

Reply:
Added in 0533c34